sqlmesh/core/model/definition.py

```diff
@@ -2203,7 +2226,16 @@ def _meta_renderer(
     "virtual_properties_": lambda value: value,
     "session_properties_": lambda value: value,
     "allow_partials": exp.convert,
-    "signals": lambda values: exp.Tuple(expressions=values),
+    "signals": lambda values: exp.tuple_(
```

Comment:
Why are we not doing the same thing as for `audits` (`_list_of_calls_to_exp`)?

Reply:
because this needs to handle the airflow syntax as well